modules/barcode-scan/src/main/resources/META-INF/substrate/config/reflectionconfig-arm64-ios.json

```diff
@@ -1,6 +1,6 @@
 [
   {
-    "name" : "com.gluonhq.attach.barcode.impl.IOSBarcodeScanService",
+    "name" : "com.gluonhq.attach.barcodescan.impl.IOSBarcodeScanService",
```

Comment:
Unrelated, can you add to the AndroidManifest in `resources/META-INF/substrate/dalvik/AndroidManifest.xml`:
```
    <uses-permission android:name="android.permission.CAMERA" />
    <application>
        <activity android:name="com.gluonhq.helloandroid.zxing.CaptureActivity"
                  android:screenOrientation="sensorLandscape"
                  android:clearTaskOnLaunch="true" android:stateNotNeeded="true"
                  android:windowSoftInputMode="stateAlwaysHidden">
             <intent-filter>
                 <action android:name="com.gluonhq.attach.barcodescan.android.SCAN"/>
                <category android:name="android.intent.category.DEFAULT"/>
             </intent-filter>
        </activity>
    </application>
```

And modify the `build.gradle` file in same folder to use Android 29. Actually there is an issue with the dependency (I'll file it).

Reply:
The `build.gradle` is already been taken care in PR #212